### ISSUE TITLE
Support for Angular 19

### ISSUE THREAD

**Issue by d-koppenhagen** [angular-eslint/angular-eslint#2117]
Since Angular 19 was released today, it should be supported by @angular-eslint/schematics as well.
This issue should track the progress.

**Comment by JamesHenry** [comment]
https://github.com/angular-eslint/angular-eslint/pull/2109

**Comment by ganatan** [comment]
**Angular installation**

npm uninstall -g @angular/cli
npm install -g @angular/cli@19.0.0
ng new angular-starter

**ESLINT Installation**

ng add @angular-eslint/schematics

**Error Message** 

✔ Determining Package Manager
  › Using package manager: npm
✔ Searching for compatible package version
  › Found compatible package version: @angular-eslint/schematics@1.0.0.
✔ Loading package information from registry
✔ Confirming installation
✔ Installing package
NOT SUPPORTED: keyword "id", use "$id" for schema ID

**Comment by ctilley83** [comment]
> #2109

Is it possible to improve release parity between angular-eslint and major Angular versions?

**Comment by kbrilla** [comment]
Any timeline on new version? Nrwl nx is waiting for this package as well as few others for Angular v19 support.  https://github.com/nrwl/nx/pull/28847

**Comment by ctilley83** [comment]
> Any timeline on new version? Nrwl nx is waiting for this package as well as few others for Angular v19 support. [nrwl/nx#28847](https://github.com/nrwl/nx/pull/28847)

If you look at #2109 it looks like it will be out any minute now. All chores are finished and checks seem to have passed.

**Comment by JamesHenry** [comment]
Yes at this stage please try out the prerelease and let me know if any blockers, otherwise I'll cut the stable by the end of the week at the absolute latest, which I don't foresee as being significantly different, only potentially some stuff to make the different package managers happy

**Comment by pfeileon** [comment]
Will this (or a near future) release include [the announced auto-migration to ESLint 9](https://github.com/angular-eslint/angular-eslint/releases/tag/v18.0.0) or should we execute `npx @eslint/migrate-config` and make all other necessary changes manually?

**Comment by JamesHenry** [comment]
@pfeileon No, I added a note about this in the release notes as well.

I haven't found `@eslint/migrate-config` to be particularly good.

The one shipped with Nx is the best one out there and if you use Nx I would recommend migrating using that tooling. Time permitting I will try and work on one for angular-eslint in a minor 19.x release
